po/LINGUAS

```diff
@@ -1,3 +1,4 @@
 # list of supported languages using locale codes
 # (i.e., the subdir names in /usr/share/locale)
 #en_US
+#ru_RU
```

Comment:
The ru_RU line needs to be uncommented.  I put "#en_US" as an example, but as a comment so it would not mess up the build.